package.json

```diff
@@ -170,25 +170,25 @@
         "@types/cytoscape-klay": "3.1.0",
         "@types/glob": "8.1.0",
         "@types/mocha": "10.0.1",
-        "@types/node": "18.x",
+        "@types/node": "20.1.0",
         "@types/prettier": "2.7.2",
-        "@types/react": "18.0.38",
-        "@types/react-dom": "18.0.11",
-        "@types/vscode": "1.77.0",
-        "@typescript-eslint/eslint-plugin": "5.59.0",
-        "@typescript-eslint/parser": "5.59.0",
+        "@types/react": "18.2.6",
+        "@types/react-dom": "18.2.4",
+        "@types/vscode": "1.78.0",
+        "@typescript-eslint/eslint-plugin": "5.59.2",
+        "@typescript-eslint/parser": "5.59.2",
         "@vitejs/plugin-react": "4.0.0",
         "@vscode/test-electron": "2.3.0",
         "autoprefixer": "^10.4.14",
         "cross-env": "^7.0.3",
-        "eslint": "8.39.0",
-        "glob": "10.2.1",
+        "eslint": "8.40.0",
+        "glob": "10.2.2",
         "less": "4.1.3",
         "mocha": "10.2.0",
         "postcss": "^8.4.23",
         "tailwindcss": "^3.3.2",
         "typescript": "5.0.4",
-        "vite": "4.3.1",
+        "vite": "4.3.5",
```

Comment:
bumping all packages because I have issues with some bleeding edge typescript features; hope bumping everything will add tooling support for what I want to use